katello/hooks/pre_validations/31-upgrade-puppet.rb

```diff
@@ -1,7 +1,7 @@
 unless app_value(:noop)
-  if File.exist?('/opt/puppetlabs/puppet/bin/ruby') && execute("rpm -q puppet-agent-oauth", false, false)
-    unless execute("/opt/puppetlabs/puppet/bin/ruby -e \"require 'oauth'\"", false, false)
-      execute("yum -y reinstall puppet-agent-oauth")
+  if File.exist?('/opt/puppetlabs/puppet/bin/ruby') && system("rpm -q puppet-agent-oauth")
+    unless system("/opt/puppetlabs/puppet/bin/ruby -e \"require 'oauth'\"")
+      Kafo::Helpers.execute("yum -y reinstall puppet-agent-oauth")
```

Comment:
Any reason you're changing this line? I thought we'd want to keep using the execute method.

Reply:
This line should just be the execute method now which I updated.